File: 70. Climbing Stairs.java
//70. Climbing Stairs


class Solution {
    public int climbStairs(int n) {
        int first=1;
        int second=1;
        for(int i=0;i<n-1;i++){
            int temp = first+second;
            first=second;
            second=temp;
        }
        return second;
    }
}

File: 70. Climbing Stairs.js
//70. Climbing Stairs


/**
 * @param {number} n
 * @return {number}
 */
var climbStairs = function(n) {
    let first=1;
    let second=1;
    for(let i=0;i<n-1;i++){
        let temp = first+second;
        first=second;
        second=temp;
    }
    return second;
};
